### PR TITLE
Improve logging functionality

### PR DESCRIPTION
**Pull request by BrynCooke**
Improve logging page
Improve subgraph URL override title.

### CODE REVIEW

File: docs/source/configuration/logging.mdx [hunk 1]
```diff
@@ -1,8 +1,29 @@
 ---
-title: Configuring Logging
-description: Tuning logging configurations
+title: Logging
+description: Logging output formats
 ---
 
 import { Link } from 'gatsby';
```
Comment (abernix) [inline docs/source/configuration/logging.mdx:7]: Should we add a section about setting the verbosity of logging?

File: docs/source/configuration/logging.mdx [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+
+```bash
+2022-03-18T10:28:11.090054Z  INFO apollo_router::executable: apollo-router@0.1.0-alpha.9
+2022-03-18T10:28:11.160650Z  INFO apollo_router: Starting Apollo Router
+2022-03-18T10:28:11.339906Z  INFO apollo_router: Listening on http://127.0.0.1:4000 🚀
+```
+
+## JSON-formatted logging:
+
+JSON-formatted logging facilitates compatibility with searchable logging facilities like Google Cloud Logging or Stackdriver. It is preferred for production environments.
```
Comment (abernix) [inline docs/source/configuration/logging.mdx:22]: How do I change it? Can I change it?  If so, we should add that here, or at least add an explanation of how we choose.

**Review by abernix — APPROVED** [review]

File: docs/source/configuration/logging.mdx (hunk 2 above)
Comment (garypen) [reply docs/source/configuration/logging.mdx:22]: You can't change it. It's auto-detected when the router starts.

File: docs/source/configuration/logging.mdx (hunk 1 above)
Comment (BrynCooke) [reply docs/source/configuration/logging.mdx:7]: Added

File: docs/source/configuration/logging.mdx (hunk 2 above)
Comment (BrynCooke) [reply docs/source/configuration/logging.mdx]: I added a sentence to state that it was the Bunyan format, and that it was popular.

**Pull request merged**